### ISSUE TITLE
Proj0219 false positive

### ISSUE THREAD

**Issue by sungam3r** [dotnet-project-file-analyzers/dotnet-project-file-analyzers#684]
Please see https://github.com/sungam3r/graphql-introspection-model/pull/246

I got `The <IsPackable> node is set to 'true' but the <OutputType> is not 'Library' (https://dotnet-project-file-analyzers.github.io/rules/Proj0219.html)` for test project. Test projects are known to have IsPackable=false due to IsTestProject=false that established by Microsoft.NET.Test.Sdk package. It's a well known behaviour in .NET. I suppressed Proj0219 for demonstration purposed and added debug message:
```
  <Target Name="AAA" BeforeTargets="Build">
    <Message Importance="high" Text="=========== IsPackable: $(IsPackable)" />
  </Target>
```

<img width="2128" height="790" alt="Image" src="https://github.com/user-attachments/assets/b09387fa-8c93-40d2-8769-4323db0085ad" />

So the statement `The <IsPackable> node is set to 'true' ` is not true.

**Comment by Corniel** [comment]
We'll try to look into this asap.
